pkg/download/bucket/bucket.go

```diff
@@ -69,17 +77,16 @@ func (d *Downloader) convertAllObjects(projectName string, objects [][]byte) []c
 
 		c, err := convertObject(o, projectName)
 		if err != nil {
-			log.WithFields(field.Type("bucket"), field.Error(err)).
-				Error("Failed to decode API response objects for bucket resource: %v", err)
-			continue
-		}
+			if errors.As(err, &skipErr{}) {
```

Comment:
Not tooooooooo happy with that, but best I've come up with. We don't need the content of the error, just need to check if it's the correct type. 
`errors.Is` does not work with this